plugin/pkg/admission/storage/persistentvolume/label/admission_test.go

```diff
@@ -369,68 +277,6 @@ func Test_PVLAdmission(t *testing.T) {
 			},
 			err: nil,
 		},
-		{
-			name:    "existing labels from user are changed",
```

Comment:
This test checks for a generic admission functionality and should be reworked to a different volume plugin. Preferably vSphere, because it will go away as the last.